### ISSUE TITLE
Move CSS to external stylesheets. Refactor codebase.

### ISSUE THREAD

**Issue by HighnessAtharva** [Not-Quite-RARBG/main#15]
- Majority of CSS is internal at this point. 
- Lot of CSS styles are inline and repeating
- I plan to move all the CSS to an external style.css for effective management and cleaner codebase

**Comment by Keklul404** [comment]
Yes, a lot of cleanup need to be made about the .css, but also about the naming of different divs (for easier maintenance in the long run)
